caddy/module.go

```diff
@@ -148,7 +148,10 @@ func (f *FrankenPHPModule) ServeHTTP(w http.ResponseWriter, r *http.Request, _ c
 		if documentRoot == "" && frankenphp.EmbeddedAppPath != "" {
 			documentRoot = frankenphp.EmbeddedAppPath
 		}
-		documentRootOption = frankenphp.WithRequestDocumentRoot(documentRoot, *f.ResolveRootSymlink)
+		//If we do not have a resolved document root, then we cannot resolve the symlink of our cwd because it may
+		//resolve to a different directory than the one we are currently in.
+		//This is especially important if there are workers running.
```

Comment:
```suggestion
		// If we do not have a resolved document root, then we cannot resolve the symlink of our cwd because it may
		// resolve to a different directory than the one we are currently in.
		// This is especially important if there are workers running.
```